src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 I downloaded from [here](https://ffmpeg.zeranoe.com/builds/) using the latest (not 3.3.3) 64-bit static, so you wouldn't complain that I used a too old version and had to try a newer.

I really don't want to do a lot of testing of parameters, please try to get them correct before asking me to test. Your latest suggestion didn't work either, as I understand the error ```hevc_cuvid``` itself doesn't support 10-bit:

```powershell
C:\Users\ffmpeg.exe -y -c:v hevc_cuvid -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -c:v h264_nvenc -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
ffmpeg version N-86986-g62dfa2b Copyright (c) 2000-2017 the FFmpeg developers
  built with gcc 7.1.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-cuda --enable-cuvid --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-nvenc --enable-avisynth --
enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-li
bfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enabl
e-libopenh264 --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolam
e --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable
-libxavs --enable-libxvid --enable-libzimg --enable-lzma --enable-zlib
  libavutil      55. 74.100 / 55. 74.100
  libavcodec     57.102.100 / 57.102.100
  libavformat    57. 76.100 / 57. 76.100
  libavdevice    57.  7.100 / 57.  7.100
  libavfilter     6. 99.100 /  6. 99.100
  libswscale      4.  7.102 /  4.  7.102
  libswresample   2.  8.100 /  2.  8.100
  libpostproc    54.  6.100 / 54.  6.100
Input #0, matroska,webm, from 'jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv':
  Metadata:
    COMPATIBLE_BRANDS: iso4hvc1iso6
    MAJOR_BRAND     : iso4
    MINOR_VERSION   : 1
    ENCODER         : Lavf56.3.100
  Duration: 00:00:30.10, start: 0.067000, bitrate: 199827 kb/s
    Stream #0:0(und): Video: hevc (Main 10), yuv420p10le(tv), 3840x2160 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 1k tbn, 29.97 tbc (default)
    Metadata:
      CREATION_TIME   : 2016-02-05 13:43:30
      LANGUAGE        : und
      HANDLER_NAME    : hevc@GPAC0.5.2-DEV-rev565-g71748d7-ab-suite
[hevc_cuvid @ 00000000004d1a80] Bit depth 10 is not supported.
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (hevc_cuvid) -> h264 (h264_nvenc))
Error while opening decoder for input stream #0:0 : Invalid argument
bench: maxrss=149092kB
```